generate.py

```diff
@@ -224,23 +415,56 @@ def merge_model_ops(src_file, dest_file):
     ]
     edit_file(f, patterns, suffix)
 
-    # 3. Merge model_variables.h into 1st project
+    # 3. Check if full tflite and make additional required modifications
+    if args.full_tflite:
+        # Insert #define EI_CLASSIFIER_USE_FULL_TFLITE 1 in model_metadata.h
+        f = os.path.join(tmpdir, p, "model-parameters/model_metadata.h")
+        insert_define_statement(f, "#define EI_CLASSIFIER_USE_FULL_TFLITE 1")
```

Comment:
This should not be inserted into model_metadata.h, but instead passed in Makefile or CMakeLists, as done here
https://github.com/edgeimpulse/example-standalone-inferencing-linux/blob/95675547fae720ab2adf7e514f89dfe231a1c0b1/Makefile#L81